packages/sui-pde/src/adapters/optimizely/index.js

```diff
@@ -46,7 +49,12 @@ export default class OptimizelyAdapter {
     this._userId = userId?.toString()
     this._activeIntegrations = activeIntegrations
     this._applicationAttributes = applicationAttributes
+
     this.updateConsents({hasUserConsents})
+    this.addNotificationListener({
```

Comment:
IMHO It could be moved and added on each website after adapter instance creation. It could be different tracking implementations (Segment, G.Analytics, etc)